lib/event.c

```diff
@@ -422,8 +422,11 @@ DEFUN_NOSH (show_event_poll,
 	struct event_loop *m;
 
 	frr_with_mutex (&masters_mtx) {
-		for (ALL_LIST_ELEMENTS_RO(masters, node, m))
+		for (ALL_LIST_ELEMENTS_RO(masters, node, m)) {
```

Comment:
Shouldn't this be changed to ALL_LIST_ELEMENTS()?

Reply:
It's a read only because we are not changing anything.  What am I missing here that we would need to change this?